webview/src/setup/state/dvcSlice.ts

```diff
@@ -0,0 +1,72 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { DvcCliDetails, SetupData } from 'dvc/src/setup/webview/contract'
+
+export type DvcState = Pick<
+  SetupData,
+  | 'canGitInitialize'
+  | 'cliCompatible'
+  | 'dvcCliDetails'
+  | 'isPythonExtensionUsed'
+  | 'projectInitialized'
+  | 'needsGitInitialized'
+  | 'pythonBinPath'
+>
```

Comment:
Should this be `Omit` instead of `Pick` (with omitted keys)? There are way more properties to pick than to omit.